packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -64,6 +64,11 @@
       String url = (String) params.get("initialUrl");
       webView.loadUrl(url);
     }
+
+    if (params.containsKey("userAgent")) {
```

Comment:
This should be done before `loadUrl` for the initial URL.

Reply:
pushed fix

Reply:
Makes sense, thanks for fixing it.